src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        //creating int with number og Figure classes to pick up one of those
+        Random randResult = new Random();
+        int figureNum = randResult.nextInt(5);
```

Comment:
```suggestion
        int figureNum = randResult.nextInt(figureCount);
```
замість 5 кращсе створити інт змінну для збільшення інформативності